ledger-api/grpc-definitions/BUILD.bazel

```diff
@@ -184,13 +184,15 @@ ledger_api_haskellpb_sources_admin = [
     "MeteringReportService.hs",
     "ObjectMeta.hs",
     "UserManagementService.hs",
+    "InspectionService.hs",
 ]
 
 genrule(
     name = "ledger-api-haskellpb-sources-admin",
     srcs = [
         "@com_google_protobuf//:well_known_protos",
         "@go_googleapis//google/rpc:status.proto",
+        ":ledger-api-protos-fg",
```

Comment:
This is now required as an admin interface depends on a non-admin proto (value.proto).